app.vue

```diff
@@ -3,10 +3,12 @@
         always visible on desktop screens without scrolling.
     -->
     <div data-testid="app" class="h-screen w-full">
-        <div v-if="store.enableModerationPanel" data-testid="moderation-content" class="h-full w-full flex flex-col font-sans text-primary-text bg-primary-bg">
+        <div v-if="store.enableModerationPanel" data-testid="moderation-content"
+            class="h-full w-full flex flex-col font-sans text-primary-text bg-primary-bg">
             <NuxtPage class="flex flex-1" />
         </div>
-        <div v-else-if="!store.enableModerationPanel && $viewport.isGreaterThan('desktop')" class="h-full w-full flex flex-col font-sans text-primary-text bg-primary-bg">
+        <div v-else-if="!store.enableModerationPanel && $viewport.isGreaterThan('desktop')"
+            class="h-full w-full flex flex-col font-sans text-primary-text bg-primary-bg">
```

Comment:
you can move the viewport check to be a css media query
like this
`portrait:hidden`

```suggestion
            class="portrait:hidden h-full w-full flex flex-col font-sans text-primary-text bg-primary-bg">
```

Reply:
you forgot to add the class 😆

Reply:
no wait, I see it on the one above 💯

Reply:
This needed to go into line 7